roles/nexus/templates/ingress.yml.j2

```
@@ -71,6 +71,6 @@ spec:
             pathType: Prefix
             backend:
               service:
-                name: nexus-docker-proxy
+                name: nexus
```

Pourquoi ce changement ?

---

initialement il y avait deux services  pour les ports exposés du container 8081 et 5000

❯ k get svc -n dso-nexus
NAME                 TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
nexus                ClusterIP   10.32.13.52    <none>        8081/TCP   62d
nexus-docker-proxy   ClusterIP   10.47.124.54   <none>        5000/TCP   62d

dans le déploiement helm : il n y a plus qu un seul service qui communique sur les deux ports
❯ k get svc -n fabrice-nexus
NAME       TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)             AGE
nexus      ClusterIP   10.37.136.51   <none>        8081/TCP,5000/TCP   17h

 A mon sens pas de nécessité de créer un deuxième service et ingress relié à cet unique service
par ailleurs on obtient en fait le meme comportement
erreur 502 bad gateway pour les deux urls

---

En effet, dans ce cas comme pour au-dessus :
```suggestion
                name: {{ dsc.nexus.release }}
```